checkstyle.xml

```diff
@@ -0,0 +1,121 @@
+            <property name="tokens" value="UNARY_MINUS"/>
+            <property name="tokens" value="UNARY_PLUS"/>
+            <property name="tokens" value="ARRAY_DECLARATOR"/>
+            <property name="tokens" value="INDEX_OP"/>
+            <property name="tokens" value="METHOD_REF"/>
+        </module>    
+        
+        <module name="EmptyBlock">
+            <property name="tokens" value="LITERAL_CATCH"/>
+            <property name="tokens" value="LITERAL_DEFAULT"/>
+            <property name="tokens" value="LITERAL_CASE"/>
+            <property name="tokens" value="INSTANCE_INIT"/>
+            <property name="tokens" value="STATIC_INIT"/>
+            <property name="tokens" value="LITERAL_DO"/>
+            <property name="tokens" value="LITERAL_ELSE"/>
+            <property name="tokens" value="LITERAL_FINALLY"/>
+            <property name="tokens" value="LITERAL_FOR"/>
+            <property name="tokens" value="LITERAL_IF"/>
+            <property name="tokens" value="LITERAL_SWITCH"/>
+            <property name="tokens" value="LITERAL_TRY"/>
+            <property name="tokens" value="LITERAL_SYNCHRONIZED"/>
+            <property name="tokens" value="LITERAL_WHILE"/>
+            <property name="option" value="text"/>
+        </module>
+ 
+        <!-- 
+        <module name="LeftCurly"/>
+        <module name="NeedBraces"/>
+        <module name="NeedBraces">
+            <property name="tokens" value="LAMBDA"/>
+            <property name="allowSingleLineStatement" value="true"/>
+        </module>
+        <module name="RightCurly">
+            <property name="tokens" value="METHOD_DEF"/>
+            <property name="tokens" value="CTOR_DEF"/>
+            <property name="tokens" value="CLASS_DEF"/>
+            <property name="tokens" value="INSTANCE_INIT"/>
+            <property name="tokens" value="LITERAL_FOR"/>
+            <property name="tokens" value="STATIC_INIT"/>
+            <property name="tokens" value="LITERAL_WHILE"/>
+            <property name="tokens" value="LITERAL_CATCH"/>
+            <property name="tokens" value="LITERAL_ELSE"/>
+            <property name="tokens" value="LITERAL_FINALLY"/>
+            <property name="tokens" value="LITERAL_IF"/>
+            <property name="tokens" value="LITERAL_TRY"/>
+            <property name="tokens" value="ANNOTATION_DEF"/>
+            <property name="tokens" value="ENUM_DEF"/>
+            <property name="tokens" value="RECORD_DEF"/>
+            <property name="tokens" value="COMPACT_CTOR_DEF"/>
+            <property name="option" value="alone"/>
+        </module>
+        <module name="RightCurly">
+            <property name="tokens" value="LITERAL_DO"/>
+            <property name="option" value="same"/>
+        </module>
+        <module name="RightCurly">
+            <property name="tokens" value="INTERFACE_DEF"/>
+            <property name="option" value="alone_or_singleline"/>
+        </module>
+        -->
```

Comment:
Can we remove this commented block or does it still hold any significance?

Reply:
This is not needed so I removed it.